CMakeLists.txt

```diff
@@ -300,6 +300,7 @@ add_executable(SystemTestsRunner
     "source/tests/system/niswitch_driver_api_tests.cpp"
     "source/tests/system/niswitch_session_tests.cpp"
     "source/tests/system/nisync_session_tests.cpp"
+    "source/tests/system/nidcpower_driver_api_tests.cpp"
```

Comment:
I think this file is already part of the features/ni-dcpower branch. I think the diff in this PR is not updated?

Reply:
Correct. I planned to change the target of this PR to feature branch. Its just that I was waiting for the current target branch to get submitted to the feature branch.

Reply:
@ganeshnie There were some merge conflicts that I could not resolve so I would close this PR and create a new PR which does the same but is directly branched out from feature branch.
New PR : [#23](https://github.com/doshirohan/grpc-device/pull/23)